src/app/pages/pages-menu.ts

```diff
@@ -32,25 +23,40 @@ export const MENU_ITEMS: NbMenuItem[] = [
     icon: 'copy-outline',
     link: '/pages/twins',
   },
-  {
-    title: 'Services',
-    icon: 'layers-outline',
-    children: [
-      {
-        title: 'LoRa',
-        icon: 'radio-outline',
-        link: '/pages/services/lora',
-      },
-      {
-        title: 'OPC-UA',
-        icon: 'globe-outline',
-        link: '/pages/services/opcua',
-      },
-      {
-        title: 'Gateways',
-        icon: 'hard-drive-outline',
-        link: '/pages/services/gateways',
-      },
-    ],
-  },
 ];
+
+if (environment.experimental) {
+  MENU_ITEMS.push(
```

Comment:
I would simply replace all Menu. With push() this is going to change the structure